src/components/Modal/README.md

```diff
@@ -547,14 +547,213 @@ export default {
+				<div
+					v-if="isActivePane(index)"
+				>
+					<img
+						:src="pane.image"
+						class="cover-photo"
+					>
+					<m-modal-content>
+						<m-heading>
+							{{ pane.heading }}
+						</m-heading>
+						<m-text>
+							{{ pane.content }}
+						</m-text>
+					</m-modal-content>
+				</div>
+			</m-transition-fade-in>
+		</div>
+		<m-inline-action-bar>
+			<m-action-bar-button
+				key="close"
+				@click="modalApi.close()"
+			>
+				<x-icon class="icon" />
+			</m-action-bar-button>
+		</m-inline-action-bar>
+	</m-modal>
+</template>
+
+<script>
+import { MHeading } from '@square/maker/components/Heading';
+import { MText } from '@square/maker/components/Text';
+import {
+ MModal, MModalContent, modalApi,
+} from '@square/maker/components/Modal';
+import { MInlineActionBar, MActionBarButton } from '@square/maker/components/ActionBar';
+import { MTransitionFadeIn } from '@square/maker/components/TransitionFadeIn';
+import XIcon from '@square/maker-icons/X';
+
+export default {
+	name: 'MultiPaneModal',
+
+	components: {
+		MModal,
+		MHeading,
+		MText,
+		MModalContent,
+		MInlineActionBar,
+		MActionBarButton,
+		MTransitionFadeIn,
+		XIcon,
+	},
+
+	inject: {
+		modalApi,
+	},
+
+	data() {
+		return {
+			activeIndex: 0,
+			multiPane: [
+				{
+					image: 'https://picsum.photos/600/300',
+					heading: 'First modal heading',
+					content: 'First modal content',
+				},
+				{
+					image: 'https://picsum.photos/400/300',
+					heading: 'Second modal heading',
+					content: 'Second modal content',
+				},
+				{
+					image: 'https://picsum.photos/300/300',
+					heading: 'Third modal heading',
+					content: 'Third modal content',
+				},
+			],
+		};
+	},
+
+	computed: {
+		shouldShowLeftArrow() {
+			return this.activeIndex > 0;
+		},
+
+		shouldShowRightArrow() {
+			return (this.activeIndex + 1) < this.multiPane.length;
+		},
+	},
+
+	methods: {
+		isActivePane(index) {
+			return this.activeIndex === index;
+		},
+
+		handleLeft() {
+			if (this.activeIndex > 0) {
+				this.activeIndex -= 1;
+			}
+		},
+
+		handleRight() {
+			if (this.activeIndex < this.multiPane.length) {
+				this.activeIndex += 1;
+			}
+		},
+	},
+};
+</script>
+
+<style scoped>
+.cover-photo {
+	width: 100%;
+	height: 600px;
+	object-fit: cover;
+	object-position: center;
+}
+
+.icon {
+	width: 24px;
+	height: 24px;
+}
+</style>
+```
+
 <!-- api-tables:start -->
+## Modal Props
+
+| Prop            | Type      | Default | Possible values | Description |
+| --------------- | --------- | ------- | --------------- | ----------- |
+| has-left-arrow  | `boolean` | `false` | —               | —           |
+| has-right-arrow | `boolean` | `false` | —               | —           |
+
+
 ## Modal Slots
 
 | Slot    | Description   |
 | ------- | ------------- |
 | default | Modal content |
 
 
+## Modal Events
+
+| Event    | Type | Description |
+| -------- | ---- | ----------- |
+| go-left  | -    | —           |
+| go-right | -    | —           |
```

Comment:
we can probably think of better names for these events, maybe just simply `left` and `right` or `navigate-left` or `navigate-right` to be more explicit/descriptive